docs/release-process.md

```diff
@@ -11,109 +11,89 @@
 | Linux ARMv7                 | _builds are provided but not tested_ |
 | Linux ARM64                 | _builds are provided but not tested_ |
 
-
 ## Release Schedule and Hotfixes
 
-Normal releases in this repository that add features such as improved RPC endpoints, improved boot-up time, new event
-observer fields or event types, etc., are released on a monthly schedule. The currently staged changes for such releases
-are in the [develop branch](https://github.com/stacks-network/stacks-core/tree/develop). It is generally safe to run
-a `stacks-node` from that branch, though it has received less rigorous testing than release tags. If bugs are found in
-the `develop` branch, please do report them as issues on this repository.
+Normal releases in this repository that add new features are released on a monthly schedule.
+The currently staged changes for such releases are in the [develop branch](https://github.com/stacks-network/stacks-core/tree/develop).
+It is generally safe to run a `stacks-node` from that branch, though it has received less rigorous testing than release tags or the [master branch](https://github.com/stacks-network/stacks-core/tree/master).
+If bugs are found in the `develop` branch, please do [report them as issues](https://github.com/stacks-network/stacks-core/issues) on this repository.
 
-For fixes that impact the correct functioning or liveness of the network, _hotfixes_ may be issued. These are patches
-to the main branch which are backported to the develop branch after merging. These hotfixes are categorized by priority
-according to the following rubric:
+For fixes that impact the correct functioning or liveness of the network, _hotfixes_ may be issued.
+These are patches to the main branch which are backported to the develop branch after merging.
+These hotfixes are categorized by priority according to the following rubric:
 
 - **High Priority**. Any fix for an issue that could deny service to the network as a whole, e.g., an issue where a particular kind of invalid transaction would cause nodes to stop processing requests or shut down unintentionally. Any fix for an issue that could cause honest miners to produce invalid blocks.
 - **Medium Priority**. Any fix for an issue that could cause miners to waste funds.
 - **Low Priority**. Any fix for an issue that could deny service to individual nodes.
 
 ## Versioning
 
-This repository uses a 5 part version number.
+This repository uses a 5 part version number:
 
 ```
 X.Y.Z.A.n
 
-X = 2 and does not change in practice unless there’s another Stacks 2.0 type event
+X major version - does not change in practice unless there’s another Stacks 2.0 type event
```

Comment:
Is this about to change to `3` for Nakamoto? If that is the case, maybe the wording could be "it only changes on ... events"

Reply:
@kantai any opinion here? I feel like we should remove the `2.0 type event` wording, but i'm unsure of what wording to use.

Reply:
Alt: "unless there's another _significant_ network upgrade (similar to Stacks 1.0 to Stacks 2.0 in scope, complexity, impact)"

And yes, I believe this should change to 3.0.x post Nakamoto.

Reply:
since this PR is meant for 3.0, i've updated in 8a8c5b09408056b533649a558c9b52fb8d44dfce